Engine/source/shaderGen/shaderGen.cpp

```diff
@@ -49,6 +49,7 @@ MODULE_BEGIN( ShaderGen )
 
 MODULE_END;
 
+String ShaderGen::smCommonShaderPath(Con::getVariable("$Core::CommonShaderPath", "shaders/common"));
```

Comment:
This will always evaluate to "shaders/common" as console subsystem won't be initialized until way later, unless it segfaults. Keep in mind when you do object initialization like this at the source file scope it'll be executed before `int main(size_t argc, char *argv[])` is executed. Shouldn't call engine functions until their respective subsystems are initialized.